docs/topics/proc_interacting-with-the-unmodified-health-check-booster.adoc

```diff
@@ -1,7 +1,8 @@
 [id='interacting-with-the-unmodified-health-check-booster_{context}']
-= Interacting with the unmodified {name-mission-health-check} booster
+[id='interacting-with-the-unmodified-health-check-example_{context}']
+= Interacting with the unmodified {name-example-health-check} example
 
-Once you have the booster deployed, you will have a service called `{value-name-app}` running that exposes the following REST endpoints:
+Once you have the example deployed, you will have a service called `{value-name-app}` running that exposes the following REST endpoints:
```

Comment:
Once you have the example deployed--> After you have the example application deployed